netci.groovy

```diff
@@ -2199,6 +2359,51 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                     // Basic archiving of the build, no pal tests
                     Utilities.addArchival(newJob, "bin/Product/**,bin/obj/*/tests/**/*.dylib,bin/obj/*/tests/**/*.so", "bin/Product/**/.nuget/**")
                     break
+                case 'arm':
+                    // Non-Windows ARM cross builds on hardware run on Ubuntu only
+                    assert (os == 'Ubuntu')
+
+                    // Debugging:
+                    // 1. show the IP address of the machine we're running on.
+                    // 2. show the directory the script is run in.
+                    // 3. show what's in this directory (currently).
+                    // 4. how much disk space is there?
+                    // etc.
```

Comment:
I understand why these things are added, however, this debugging information can be used for nefarious purposes. I would prefer to just limit this to printing a hostname, printing even the current directory and filesystem contents sounds unsafe.